packages/nodes-base/nodes/Airtop/actions/file/helpers.ts

```diff
@@ -252,9 +252,11 @@ export async function triggerFileInput(
 	fileId: string,
 	windowId: string,
 	sessionId: string,
+	elementDescription = '',
```

Comment:
Should this be `elementDescription?: string,` instead? That way it defaults to undefined vs ''?